packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -305,13 +306,52 @@ Object.keys(localStorage).forEach(key => {
 })
 
 function ConnectedChainSyncer() {
+  const { isSmartContractWallet, isLoading } = useAccountType()
```

Comment:
NIT: Can we rename this instance of `isLoading` to `isAccountTypeLoading` so that it's easier to read in the code that follows?